src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -1008,7 +1020,7 @@ export default {
 				document.querySelector('#header'),
 			], {
 				allowOutsideClick: true,
-				fallbackFocus: this.$refs.closeButton.$el,
+				fallbackFocus: this.$refs.closeButton?.$el ?? this.$refs.sidebar,
```

Comment:
It is always available 

```suggestion
				fallbackFocus: this.$refs.closeButton.$el,
```